common/src/db/query/columns.rs

```diff
@@ -190,12 +158,18 @@ impl Columns {
         value: &str,
     ) -> Result<SimpleExpr, Error> {
         self.for_field(field).and_then(|(lhs, ct)| {
-            match (value.to_lowercase().as_str(), operator) {
-                ("null", Operator::Equal) => Ok(lhs.is_null()),
-                ("null", Operator::NotEqual) => Ok(lhs.is_not_null()),
-                (_, Operator::Like) => Ok(lhs.ilike(like(value))),
-                (_, Operator::NotLike) => Ok(lhs.not_ilike(like(value))),
-                _ => parse(value, &ct).map(|rhs| lhs.binary(operator, rhs)),
+            match (value.to_lowercase().as_str(), operator, &ct) {
+                ("null", Operator::Equal, _) => Ok(lhs.is_null()),
```

Comment:
I know the PR doesn't change this, but this feels like a massive mistake: https://www.wired.com/story/null-license-plate-landed-one-hacker-ticket-hell/